services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/nodeports.py

```diff
@@ -33,7 +34,16 @@
 
 def _get_size_of_value(value: ItemConcreteValue) -> int:
     if isinstance(value, Path):
-        size_bytes = value.stat().st_size
+        # if symlink we need to fetch the pointer to the file
+        # relative symlink need to know which their parent is
+        # in oder to properly resolve the path since the workdir
+        # does not equal to their parent dir
+        path = (
+            Path(value.parent) / Path(os.readlink(value))
+            if value.is_symlink()
+            else value
+        )
```

Comment:
```suggestion
        path = value
        if value.is_symlink():
          path = Path(value.parent) / Path(os.readlink(value))        
```
maybe a bit more readable and you can spare the long comments above?